test/parallel/test-tls-over-http-tunnel.js

```diff
@@ -41,7 +41,8 @@ var proxy = net.createServer(function(clientSocket) {
       // Verify the CONNECT request
       assert.equal('CONNECT localhost:' + common.PORT + ' HTTP/1.1\r\n' +
                    'Proxy-Connections: keep-alive\r\n' +
-                   'Host: localhost:' + proxyPort + '\r\n\r\n',
+                   'Host: localhost:' + proxyPort + '\r\n' +
+                   'Connection: close\r\n\r\n',
```

Comment:
Maybe, but that is not really relevant to this change. The `Proxy-Connections` header is just like a custom http header and does not mean anything in this context. The plan was not do to unrelated changes in the tests.